### PR TITLE
web/marketing/pages: partner UI table

### PR DESCRIPTION
**Pull request by fuskovic**
What: Adds the markup and styling to the admin portal to display the offers table for partner offers to the UI.

Why: So the admin can have a visual of partner offers.

Please describe the tests:
 - Test 1: none needed / markup and styling only
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: web/marketing/pages/partner-offers-modal.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                            <input type="text" class="form-control" name="Description" id="Description" placeholder="Our test with $50 for May" required>
+                        </div>
+                        <div class="form-group col-md-4">
+                            <label for="ExpiresAt">Credit Exp Date</label>
+                            <input type="date" class="form-control" id="ExpiresAt" name="ExpiresAt" placeholder="06/01/19" required>
+                        </div>
+                    </div>
+                    <div class="form-row">
+                        <div class="form-group col-md-2">
+                            <label for="InviteeCredit">Give Credit</label>
```
Comment (VinozzZ) [inline web/marketing/pages/partner-offers-modal.html]: I don't think we need this column for partner offer

File: web/marketing/pages/partner-offers-modal.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                            <input type="number" class="form-control" name="InviteeCredit" id="InviteeCredit" placeholder="$50" required>                        
+                        </div>
+                        <div class="form-group col-md-2">
+                            <label for="InviteeCreditDurationDays">Give Credit Exp.</label>
```
Comment (VinozzZ) [inline web/marketing/pages/partner-offers-modal.html]: same as above

File: web/marketing/pages/partner-offers-modal.html (hunk 1 above)
Comment (fuskovic) [reply web/marketing/pages/partner-offers-modal.html]: Just to confirm based on our stand up today, we are keeping this and removing the AwardCredit for the partner `offerType` correct ?

Comment (VinozzZ) [reply web/marketing/pages/partner-offers-modal.html]: Yes

**Pull request merged**